src/index.tsx

```diff
@@ -4,11 +4,14 @@ import {Provider} from 'react-redux';
 import App from './App';
 import './index.css';
 import reportWebVitals from './reportWebVitals';
-import store from './store';
+import {store, persistor} from './store';
+import {PersistGate} from 'redux-persist/integration/react';
```

Comment:
Move this import to the top. The import order generally will be
```typescript
// `@material-ui` things
import React from 'react';
// Any other 3rd-party imports
// Internal imports
```

Reply:
Fixed